docs/releases/2.1.8.txt

```diff
@@ -9,4 +9,5 @@ Django 2.1.8 fixes a bug in 2.1.7.
 Bugfixes
 ========
 
-*
+* Prevented admin inlines for auto-created ``ManyToManyField``\s from being
+  editable if the user only has the view permission (:ticket:`30289`).
```

Comment:
`only has` -> `only has`?

Reply:
@felixxm: what was the suggested change here?

Reply:
ahh, `only has` -> `has only` :)

Reply:
either could be correct

Reply:
Just checking: is it not the _through model_ that's auto-created in this case? (Even though the docs for auto_created say, `indicates if the field was automatically created`.)

If so maybe:

    Prevented admin inlines for``ManyToManyField``\s without explicit through
    models from being editable if the user only has the view permission
    (:ticket:`30289`).

?

Here `without explicit through models` could be `with auto-created through models` but I don't like that as much.